src/module/io.c

```diff
@@ -532,7 +532,13 @@ void stdinIsTerminal(WrenVM* vm)
 
 void stdoutFlush(WrenVM* vm)
 {
-  fflush(stdout);
+  int result = fflush(stdout);
+  if (result != 0) {
+    wrenSetSlotString(vm, 0, "Cannot flush Stdout.");
+    wrenAbortFiber(vm, 0);
+    return;
+  }
+
   wrenSetSlotNull(vm, 0);
```

Comment:
@ruby0x1 Is this a case where we need `wrenEnsureSlots(vm, 1);` or can it be assumed that there is always room for at LEAST the return value in slot 0?

Reply:
if you're setting a slot, you should use ensureSlots unconditionally

Reply:
Ok, well there is another PR. :)